content/tutorials/quickstart/create-trustline-send-currency.md

```diff
@@ -53,7 +53,8 @@ To create a trustline between accounts:
 
 3. Enter a [currency code](https://www.iban.com/currency-codes) in the **Currency** field.
 4. Enter the maximum transfer limit in the **Amount** field.
-5. Click **Create Trustline**.
+5. Enter the destination account value in the **Destination** field.
+6. Click **Create Trustline**.
```

Comment:
Per our style guide, "trust line" is two words. (The corresponding UI element should also change.)